backend/tests/userTests.test.js

```diff
@@ -0,0 +1,89 @@
+// M6 Q3
+//import {jest} from '@jest/globals';
+//import axios from 'axios';
+var axios = require('axios');
+const { ObjectID } = require('bson');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+jest.mock('axios');
+
+//UserStore.js tests
+
+test('Get user tokens', () => {
+    const req = '1, 2, 3';
+    const res = [{'userID': 1, 'token': 'a'}, {'userID': 2, 'token': 'b'}, {'userID': 3, 'token': 'c'}];
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/getUserTokens?userids=' + req).then(response => expect(response).toEqual(res))
+});
+
+
+test('Store user token', () => {
+    const req = {'userID': 1, 'token': 'a'};
+    const res = {'result': 'Sucessfully stored user token'};
+
+    axios.post = jest.fn().mockResolvedValue(res);
+    return axios.post(UserStoreURL + '/storeUserToken', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Check user exists', () => {
+    const req = 'test@test.com';
+    const res = {'userID': 1};
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserStoreURL + '/checkUserExists?email=' + req).then(response => expect(response).toEqual(res))
+});
+
+//UserDB.js tests
+
+test('Scan DB for user email', () => {
+    const req = 'test@test.com';
+    const res = {"userID": 1};
+
+    axios.get = jest.fn().mockResolvedValue(res);
+    return axios.get(UserDBURL + '/scanDB?email=' + req).then(response => expect(response).toEqual(res))
+});
+
+test("Store user info", () => {
+  const req = { email: "test4@test.com" };
+    const res = {'userID': 4};
+
+    axios.post = jest.fn().mockResolvedValue(res);
+    return axios.post(UserDBURL + '/storeUserInfo', {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(req)
+    }).then(response => expect(response).toEqual(res));
+});
+
+test('Store token', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'Store·token'` with `"Store·token"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Store token", () => {
```